### PR TITLE
exiftool: add print example and image metadata example

### PR DESCRIPTION
**Pull request by mattdutson**
The second new example shows how to remove most image metadata (e.g., geolocation, camera model) without messing up the image's appearance.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/exiftool.md [hunk 1]
```diff
@@ -3,9 +3,17 @@
...
 > More information: <https://exiftool.org>.
 
+- Print the EXIF metadata for a given file:
+
+`exiftool {{file}}`
+
 - Remove all EXIF metadata from the given files:
 
-`exiftool -All= {{file1 file2 ...}}`
+`exiftool -All= {{file1}} {{file2}} {{file3}}`
```
Comment (kbdharun) [inline pages/common/exiftool.md:12]:
```suggestion
`exiftool -All= {{file1 file2 ...}}`
```
Revert this change. As this is what we use as a placeholder for multiple files. Check out our [style-guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md) to know more.

https://github.com/tldr-pages/tldr/blob/d6673b79b546780e485fddd3c4287b7cbc4177a8/contributing-guides/style-guide.md?plain=1#L148

File: pages/common/exiftool.md [hunk 2]
```diff
@@ -3,9 +3,17 @@
...
+
+- Remove all EXIF metadata from the given image files, then re-add metadata for color and orientation:
+
+`exiftool -All= -tagsfromfile @ -colorspacetags -orientation {{image1}} {{image2}} {{image3}}`
```
Comment (kbdharun) [inline pages/common/exiftool.md:16]:
```suggestion
`exiftool -All= -tagsfromfile @ -colorspacetags -orientation {{image1 image2 ...}}`
```
Same here

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by mattdutson** [comment]
Thanks for the feedback. I’ve incorporated your suggested changes.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]
